zcash_primitives/CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ and this library adheres to Rust's notion of
 
 ### Changed
 - MSRV is now 1.51.0.
+- Bumped dependencies to `ff 0.11`, `group 0.11`, `bls12_381 0.6`, `jubjub 0.8`.
```

Comment:
```suggestion
- Bumped dependencies to `ff 0.11`, `group 0.11`, `bls12_381 0.6`,
  `jubjub 0.8`, `pasta_curves 0.2`
```

Reply:
It's fine as-is; it's within 80 characters, which is what this time is wrapped at IIRC.

Reply:
Right, but `pasta_curves` was bumped.

Reply:
We haven't cut a release containing Orchard logic in a public API yet, so bumping the `pasta_curves` crate doesn't change anything.